api/omnibox/input-suggestions/background.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// This event is fired each time the user updates the text in the omnibox,
+// as long as the extension's keyword mode is still active.
+chrome.omnibox.onInputChanged.addListener((text, suggest) => {
+  console.log('inputChanged: ' + text);
+  suggest([
```

Comment:
I don't see these suggestions appear anywhere.

Reply:
I'd actually like to have the opening behavior specifically look for these entries and do something special based on whichever one you choose.

Reply:
> I don't see these suggestions appear anywhere.

This is caused by a Chrome bug. This API is partially implemented by JS in Chrome and that JS tries to use DOMParser. Since service workers don't have access to DOM related capabilities, this causes Chrome to throw.

I think this bug blocks merging this PR as the example isn't usable in it's current form. I'll add a note about this to the main PR comment. 

> I'd actually like to have the opening behavior specifically look for these entries and do something special based on whichever one you choose.

Off the top of my head one example might be to open a Google search term in the current window, a new tab, or a new window. I'll run with that for now, but if you (or anyone else) have a better idea I'm happy to do something else.